docs/spanish/README.md

```diff
@@ -33,7 +33,7 @@ Este directorio contiene toda la documentación sobre cómo contribuir a freeCod
 1. Guía de estilo sobre cómo crear Guías.
 2. Guía de estilo sobre cómo crear Desafíos de programación.
 
-## Referencía rápida de comandos para trabajar en local
+## Referencia rápida de comandos para trabajar en local
```

Comment:
Did you find any other issues on this file?

Reply:
No, this was the only mistake on the file